dashboard/src/components/notification/application-notifications.factory.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2015-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+'use strict';
+
+
+/**
+ * This class is handling all application notifications.
+ *
+ * @author Ann Shumilova
+ */
+export class ApplicationNotifications {
+
+  /**
+   * Default constructor that is using resource
+   * @ngInject for Dependency injection
+   */
+  constructor ($http) {
+    this.$http = $http;
+
+    this.notifications = [];
+  }
+
+  /**
+   * Returns the list of application notification.
+   * Notification has title, content and type (info, error, warning).
+   *
+   * @returns {*} the list of application notifications
+   */
+  getNotifications() {
+    return this.notifications;
+  }
+
+  /**
+   * Add new notification.
+   *
+   * @param notification notification yo be added
+   */
+  addNotification(notification) {
+    this.notifications.push(notification);
+  }
+
+  /**
+   * Add error notification.
+   *
+   * @param title notification's title
+   * @param content notification's content
+   * @returns {{notification}} notification
+   */
+  addErrorNotification(title, content) {
```

Comment:
maybe we could factorize code of addErrorNotification, addWarningNotification and addInfoNotification as only type is changing